ci-cd-bootstrap/namespaces/labs-ci-cd.yml

```diff
@@ -0,0 +1,15 @@
+---
+kind: List
```

Comment:
Just following the examples I was provided by @oybed . I see no problem with unwrapping.